packages/types/src/span.ts

```diff
@@ -54,6 +55,9 @@ export interface SpanJSON {
   trace_id: string;
   origin?: SpanOrigin;
   _metrics_summary?: Record<string, Array<MetricSummary>>;
+  profile_id?: string;
```

Comment:
@jjbayer just to check if this is technically correct: Can we send these fields for a `span` both in the (current) span envelope, as well as for a span in a transaction event?

Reply:
To be clear we wont actually set this ever for spans in a transaction event, but the types for these are shared right now...

Reply:
> Can we send these fields for a span both in the (current) span envelope, as well as for a span in a transaction event?
Yes.